js/homework.js

```diff
@@ -0,0 +1,157 @@
+    var green1 = document.querySelector("#bikeButton.btn-primary");
+    var green2 = document.querySelector("#volunteerButton.btn-secondary");
+
+    if (!green.className.includes("green")) {
+        green.className += " green";
+    }
+    if (!green1.className.includes("bikeButtonBlack")) {
+        green1.className += " bikeButtonBlack";
+    }
+    if (!green2.className.includes("volunteerButtonGreen")) {
+        green2.className += " volunteerButtonGreen";
+    }
+}
+/*
+var submitBtn = document.getElementById('submitBtn');
+submitBtn.addEventListener("click", checkEmail);
+
+function checkEmail(evt) {
+    evt.preventDefault();
+    var inputEmail = document.getElementById("exampleInputEmail1");
+    var email = false
+
+    if (inputEmail && inputEmail.value.indexOf("@") !== -1) {
+        return true;
+    }
+    else if (!email) {
+        alert('Please provide a valid email address');
+        inputEmail.style.backgroundColor = '##F4FA58';
+        inputEmail.focus;
+    }
+    else if (inputEmail.value == "") {
+        alert("Error: Email is empty!");
+        inputEmail.style.backgroundColor = '##F4FA58';
+        inputEmail.focus();
+        return false;
+    }
+
+}
+
+var submitBtn = document.getElementById('submitBtn');
+submitBtn.addEventListener("click", checkName);
+
+function checkName(evt) {
+    evt.preventDefault();
+    var inputText = document.getElementById("example-text-input");
+
+    if (inputText.value == "") {
+        alert("Error: Name is empty!");
+        inputText.style.backgroundColor = '#F4FA58';
+        inputText.focus();
+        return false;
+    }
+}
+
+var submitBtn = document.getElementById('submitBtn');
+submitBtn.addEventListener("click", checkYou);
+
+function checkYou(evt) {
+    evt.preventDefault();
+    var inputTextArea = document.getElementById("exampleTextarea");
+
+    if (inputTextArea.value == "") {
+        alert("Error: Describe yourself field is empty!");
+        inputTextArea.style.backgroundColor = '#F4FA58';
+        inputTextArea.focus();
+        return false;
+    }
+}
+*/
+var submitBtn = document.getElementById('submitBtn');
+submitBtn.addEventListener("click", checkForm);
+
+function checkForm() {
+    event.preventDefault();
+    var inputName = document.getElementById("example-text-input");
+    var inputEmail = document.getElementById("exampleInputEmail1");
+    var inputTextArea = document.getElementById("exampleTextarea");
+    var email = false
+
+    if (inputEmail && inputEmail.value.indexOf("@") !== -1) {
+        email = true;
+    }
+    else if (!email) {
+        alert('Please provide a valid email address');
+        inputEmail.style.backgroundColor = 'red';
+        inputEmail.focus;
+    }
+    else if (inputEmail.value == "") {
+        alert("Error: Email is empty!");
+        inputEmail.style.backgroundColor = 'red';
+        inputEmail.focus();
+        return false;
+    }
+    if (inputName.value == "") {
```

Comment:
Might be best to split this in to smaller functions as the code looks pretty similar to the following others. 

The only ones that is different is the email that has got and extra validation required. And example of that function could be:
`function validateNotEmpty(field, alertTxt) {
  if (field.value == "") {
    alert(alertTxt);
    field.style.backgroundColor = 'red';
    field.focus();
    return false;
  }   
  return true;
}
`

Then the following lines do compress to:
`if (validateNotEmpty(inputName, "Error: Name is empty!") && 
     validateNotEmpty(inputTextArea, "Error: Describe yourself field is empty!"){
        alert('Thanks for submitting your information');
        document.getElementById("contact-form").reset()
        inputTextArea.style.backgroundColor = '';
        return true;
}`